standalone.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const SerializerSelector = require('./index')
+
+function StandaloneSerializer (options = { readMode: true }) {
+  if (options.readMode === true && !options.restoreFunction) {
+    throw new Error('You must provide a restoreFunction options when readMode ON')
```

Comment:
a options  (singular article for plural word)
maybe:
```suggestion
    throw new Error('You must provide a function for the restoreFunction-option when readMode ON')
```